docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,49 @@
+# AntennaAttributionDocumentReporter
+
+The AntennaAttributionDocumentReporter generates an attribution document in PDF format by leveraging the [Eclipse 
+Antenna][1] project. The attribution document can be customized by providing a template bundle JAR, or a directory with 
+template PDF files. 
+There is a guide in the [Antenna project documentation][2] which describes how such a template bundle JAR file for the 
```

Comment:
'There is a guide in the [Antenna project documentation][2] which describes how such a template bundle JAR file for the  customization has to be created."  => better and shorter: "See [this guide][2] to learn how to create a template bundle JAR file."

Reply:
Done.